src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -567,7 +567,6 @@
     <Reference Include="System.Reflection.Emit.Lightweight" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\System.Threading.Tasks.Extensions\src\System.Threading.Tasks.Extensions.csproj" />
```

Comment:
Curious if this change is necessary or just cleanup.

Reply:
it was necessary earlier as i was building this project for netcoreapp3.0(which shouldnt have been done). now its just a cleanup.